sdlib/d/gc/extent.d

```diff
@@ -482,6 +482,21 @@ public:
 		return *_gcMetadata.outlineBitmap;
 	}
 
+	ulong scanningMarkBitsSparse(ubyte gcCycle) {
```

Comment:
This name is unfortunate. It is named after what uses the method rather than what it does.

This is guaranteed nonsense after refactoring, or more uses are added.

Reply:
I'm unsure how to respond to this. It's getting the marking bits from the extent. It's doing it for a sparse extent.

Is the problem just that the large/sparse functions are combined into one?

Reply:
`It's getting the marking bits from the extent` so why is it called scanningSomething?

Reply:
Updated